### DISCUSSION TITLE
ChatGPT chat history import issue

### DISCUSSION THREAD

**Discussion by Tostino** [open-webui/open-webui#5205] [General]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.3.19
- **Ollama (if applicable):** N/A

- **Operating System:** Pop_OS!
- **Browser (if applicable):** FireFox (latest)

**Confirmation:**

- [X ] I have read and followed all the instructions provided in the README.md.
- [X ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

You import your chat history and the chat contents, including branches, timestamps, etc properly make it into the OpenWebUI structure

## Actual Behavior:

The conversation branches are missing and messed up in the conversation list, the timestamps are thrown away, and the model name that you had the conversation with is lost.

## Description

**Bug Summary:**
See above.

## Reproduction Details

**Steps to Reproduce:**
Export your data from ChatGPT, and import it into OpenWebUI using the settings -> chat menu.

## Logs and Screenshots

**Browser Console Logs:**
No errors

**Docker Container Logs:**
No errors

**Screenshots/Screen Recordings (if applicable):**
Messed up conversation with multiple branches:
![image](https://github.com/user-attachments/assets/fb0e79bd-af81-46cb-97c7-aec5a7860d7c)

**Notes:**
Claude chat history is also even more broken.

**Comment by Tostino** [comment]
Why was this converted to a discussion? It is a broken feature.

**Reply by meganoob1337** [reply]
i dont think this is a broken Feature as the Feature doesn't Exist. I only see the Feature to import Chat History from OWUI itself, maybe thats why its a discussion

**Comment by agibson-fl** [comment]
Adding a peripheral issue to this discussion, I was able to import some simple chats from chatgpt but they're all dated "Today" rather than the date of the original chat in chatgpt.

**Comment by stephenjoly** [comment]
I still have this same issue. I also lost all my users when importing the config

**Comment by glenviewjeff** [comment]
failed for me as well, many missing chats. Is there no bug report?

**Comment by meganoob1337** [comment]
is it correct, that the import from Chatgpt isn't even a Feature so it probably is not technically a bug right?

I Found no Documentation to import chat history from Chatgpt , only import exports from open-webui itselft.

It Would Probably be a nice idea to create a Feature Request  or that somebody implements a import chatgpt history function in a PR.

Or just some script to import them somehow :D

**Reply by nikhilmaddirala** [reply]
Yes, someone should post this as a feature request

**Reply by Classic298** [reply]
correct there is no such feature.

you could write a script though that re-formats the chatgpt conversations into open webui's format.
